interface/patient_file/pos_checkout_ippf.php

```diff
@@ -0,0 +1,2798 @@
+            foreach ($taxarr as $taxlineid => $tax) {
+                if ($taxlineid === $lineid) {
+                    $aTaxes[$taxid] += $tax;
+                    $totlinetax += $tax;
+                    $aInvTaxes[$taxid][$taxlineid] = 0;
+                }
+            }
+        }
+    }
+    // $aTaxes now contains the total of each tax type (keyed on tax ID) for this line item,
+    // and those matched amounts are removed from $aInvTaxes.
+    return $totlinetax;
+}
+
+// Output HTML for a receipt line item.
+//
+function receiptDetailLine(
+    $code_type,
+    $code,
+    $description,
+    $quantity,
+    $charge,
+    &$aTotals = '',
+    $lineid = '',
+    $billtime = '',
+    $postdate = '',
+    $chargecat = ''
+) {
+    global $details, $TAXES_AFTER_ADJUSTMENT;
+
+    // Use $lineid to match up (and delete) entries in $aInvTaxes with the line.
+    $aTaxes = array();
+    $totlinetax = pull_tax($lineid, $aTaxes);
+    // $aTaxes now contains the total of each tax type for this line item, and those matched
+    // amounts are removed from $aInvTaxes.
+
+    $adjust = 0;
+    $memo = '';
+    $isadjust = false;
+
+    // If an adjustment, do appropriate interpretation.
+    if ($code_type === '') {
+        $isadjust = true;
+        $adjust = 0 - $charge;
+        $charge = 0;
+        list($payer, $code_type, $code) = explode('|', $code);
+        $memo = $description;
+        $description = $GLOBALS['simplified_demographics'] ? '' : "$payer ";
+        $description .= $code ? xl('Item Adjustment') : xl('Invoice Adjustment');
+        $quantity = '';
+    } else {
+        // Total and clear adjustments in $aAdjusts matching this line item.
+        $adjust += pull_adjustment($code_type, $code, $billtime, $memo);
+    }
+
+    $charge = formatMoneyNumber($charge);
+    $total = formatMoneyNumber($charge + $totlinetax - $adjust);
+    if (empty($quantity)) {
+        $quantity = 1;
+    }
+    $price = formatMoneyNumber($charge / $quantity, 2);
+    $tmp = formatMoneyNumber($price);
+    if ($price == $tmp) {
+        $price = $tmp;
+    }
+    if (is_array($aTotals)) {
+        $aTotals[0] += $quantity;
+        $aTotals[1] += $price;
+        $aTotals[2] += $charge;
+        $aTotals[3] += $adjust;
+        $aTotals[4] += $total;
+        // Accumulate columns 5 and beyond for taxes.
+        $i = 5;
+        foreach ($aTaxes as $tax) {
+            $aTotals[$i++] += $tax;
+        }
+    }
+
+    if (!$details) {
+        return;
+    }
+    if (empty($postdate) || substr($postdate, 0, 4) == '0000') {
+        $postdate = $billtime;
+    }
+    echo " <tr>\n";
+    echo "  <td title='" . xla('Entered') . ' ' .
+         text(oeFormatShortDate($billtime)) . attr(substr($billtime, 10)) . "'>" .
+         text(oeFormatShortDate($postdate)) . "</td>\n";
+    echo "  <td>$code</td>\n";
+    echo "  <td>$description</td>\n";
+    echo "  <td class='text-center'>" . ($isadjust ? '' : $quantity) . "</td>\n";
```

Comment:
text() on $code, $description, $quantity and around all the oeFormatMoney() calls below